ansible/roles/tenderdash/templates/docker-compose.yml.j2

```diff
@@ -0,0 +1,12 @@
+version: '3'
+
+services:
+  tenderdash:
+    image: {{ tendermint_image }}
+    container_name: tenderdash
+    entrypoint: sh -c "/usr/bin/tenderdash node"
```

Comment:
Why do we need this?

Reply:
I don't know, I just replicated the old config method

Reply:
We don't have such thing in dashmate. We don't need this anymore.